src/vs/base/parts/tree/browser/treeDefaults.ts

```diff
@@ -91,8 +91,11 @@ export enum ClickBehavior {
 	ON_MOUSE_UP
 }
 
+export type WorkbenchOpenMode = 'singleClick' | 'doubleClick';
```

Comment:
in that case, the configuration should be renamed as well? At the moment, it's named as `workbench.openMode`